lib/std/js/jsasynchttpclient.nim

```diff
@@ -54,40 +54,33 @@ runnableExamples("-d:nimExperimentalJsfetch -r:off"):
   from std/jsfetch import Response
   from std/uri import parseUri, Uri
 
-  let client: JsAsyncHttpClient = newJsAsyncHttpClient()
-  const data: string = """{"key": "value"}"""
-
-  block:
-    proc example() {.async.} =
-      let url: Uri = parseUri("http://nim-lang.org")
-      let content: cstring = await client.getContent(url)
-      let response: Response = await client.get(url)
-    discard example()
-
-  block:
-    proc example() {.async.} =
+  proc example() {.async.} =
+    let client: JsAsyncHttpClient = newJsAsyncHttpClient()
+    const data: string = """{"key": "value"}"""
```

Comment:
here and everywhere else: don't put redundant types; it's not common to do and it's not needed, the types are easily accessible (eg by reading at declaration of newJsAsyncHttpClient)

future nim versions might even generate hyperlinks automatically to jump to declaration

in short:
```
let content: cstring = await client.getContent(url)
=>
let content = await client.getContent(url)
```
etc